### PR TITLE
Add setPzLocked spell function

### PR DESCRIPTION
**Pull request by dudantas**
# Description
Work with: https://github.com/opentibiabr/otservbr-global/pull/2588

### CODE REVIEW

File: src/lua/scripts/luascript.cpp [hunk 1]
```diff
@@ -17999,6 +17999,19 @@ int LuaScriptInterface::luaSpellVocation(lua_State* L)
 	return 1;
 }
 
+int LuaScriptInterface::luaSpellSetPzLocked(lua_State* L)
+{
+	// spell:setPzLocked(bool)
+	Spell* spell = getUserdata<Spell>(L, 1);
+	if (spell) {
+		pushBoolean(L, spell->setPzLocked(getBoolean(L, 2));
```
Comment (beats-dh) [inline src/lua/scripts/luascript.cpp]: One is missing ) after the 2. But when trying to compile we get the error " Cannot convert an argument 2 from 'void' to 'bool' "

File: src/lua/scripts/luascript.cpp [hunk 2]
```diff
@@ -17999,6 +17999,19 @@ int LuaScriptInterface::luaSpellVocation(lua_State* L)
...
+		}
```
Comment (beats-dh) [inline src/lua/scripts/luascript.cpp]: To remove }

**Review by beats-dh — CHANGES_REQUESTED** [review]

**Comment by Beats-Dh** [comment]
in runes is ```rune:setPzLocked()``` takes that add tbm in src?

**Comment by dudantas** [comment]
https://github.com/Beats-Dh/otservbr-global/pull/51

**Pull request closed without merging**